CONTRIBUTING.md

```diff
@@ -17,18 +15,19 @@ isolate the function(s) that cause breakage, submit matched and complete C++
 and Python snippets that can be easily compiled and run on my end.
```

Comment:
Replace the last part of the sentence with "... compiled and run in isolation"?
Mainly because "my" isn't correct anymore.